ui/src/router/index.ts

```diff
@@ -45,6 +46,11 @@ const router = createRouter({
         else whenever(rolesAreLoaded, () => checkRoles())
       },
     },
+    {
+      path: '/configuration',
+      name: 'Configuration',
+      component: ProvisionDConfig
```

Comment:
Is this a high traffic interface that we should include in the main bundle or can we code split this like most of the other routes below? Eg. ` component: () => import('@/components/Map/MapAlarmsGrid.vue')`. It would keep our initial bundle smaller.